packages/ui/Placeholder/tests/PlaceholderControllerDecorator-specs.js

```diff
@@ -0,0 +1,29 @@
+import '@testing-library/jest-dom';
+import {render} from '@testing-library/react';
+
+import {PlaceholderControllerDecorator} from '../../Placeholder';
+
+describe('PlaceholderControllerDecorator', () => {
+	let data;
+
+	const DivComponent = (props) => {
+		data = props;
+
+		return <div data-testid="component" />;
+	};
+
+	const Component = PlaceholderControllerDecorator({notify: 'onScroll'}, DivComponent);
```

Comment:
```suggestion
	const Component = PlaceholderControllerDecorator(DivComponent);
```
Since `notify: 'onScroll'` is defaultConfig, I think we can just past DivComponent to test if it pass onScroller handler.